packages/terra-functional-testing/src/config/utils/getDefaultThemeName.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const path = require('path');
+
+const CONFIG = 'terra-theme.config.js';
+
+module.exports = () => {
+  let themeConfig = {};
+
+  const defaultConfig = path.resolve(process.cwd(), CONFIG);
+  if (fs.existsSync(defaultConfig)) {
+    // eslint-disable-next-line global-require, import/no-dynamic-require
+    themeConfig = require(defaultConfig);
+  }
+  return themeConfig.theme || 'terra-default-theme';
```

Comment:
if theme isn't defined we use  `terra-default-theme` for the theme name

Reply:
I think we can also remove [this default theme](https://github.com/cerner/terra-toolkit/blob/cb967914cd9f14eeb54d7bf9c1847cc6592d366c/packages/terra-functional-testing/src/services/wdio-terra-service.js#L26) in wdio-terra-service. I don't think it was needed without this change since we're were defaulting it in [test runner](https://github.com/cerner/terra-toolkit/pull/623/files#diff-b821c3a083e050c54034354280dac343f0b8d9ad66c460cad80a99ec7ebea640) before.

Reply:
updated in: https://github.com/cerner/terra-toolkit/pull/623/commits/5d59c66b4cea90a2d10813c8c0f8ced6fb30cf64